tests/mamba/test_selective_state_update_mtp.py

```
@@ -440,9 +398,37 @@ def run_kernel_with_intermediate_states(self, inputs, out=None):
+        dstate,
+        cache_steps,
+        state_dtype,
+        weight_dtype,
+        use_out_tensor,
+    ):
         """Test that kernel output matches and intermediate states are cached correctly."""
-        y_ref, state_ref, intermediate_states_ref = reference_output
+        inputs = self.make_inputs(
+            batch, nheads, dim, dstate, cache_steps, state_dtype, weight_dtype
+        )
+        y_ref, state_ref, intermediate_states_ref = self.make_reference_output(inputs)
```

Ruff (0.15.1)

[warning] Unpacked variable `state_ref` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)